src/01-general/work-orders.apib

```diff
@@ -0,0 +1,199 @@
+## Work Orders [/workOrders]
+
+### workOrders.draft [POST /workOrders.draft]
+
+Add a draft work order.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (WorkOrderData)
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (object)
+            + type: `workOrder` (string)
+            + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string)
+
+### workOrders.update [POST /workOrders.update]
+
+Update a draft work order.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string)
+        + Include WorkOrderData
+
++ Response 204
+
+### workOrders.drafts [GET /workOrders.drafts]
```

Comment:
PP : I don't think we need a separate endpoint for that

Reply:
but in that case we will need to have a "state" field on the workorder and a filter for it

Reply:
I just assume we will have the need of showing combined lists of draft and finalized work orders...

Reply:
To be discussed in meeting.

Reply:
Distinct endpoint for drafts will go and a state property + filter on `.list` will be added.